pennylane_lightning/core/src/simulators/lightning_qubit/measurements/tests/Test_MeasurementsLQubit.cpp

```diff
@@ -521,6 +521,84 @@ TEMPLATE_PRODUCT_TEST_CASE("Probabilities", "[Measurements]",
                                        .margin(1e-7));
             }
         }
+        SECTION("5 qubits") {
```

Comment:
We could combine many cases above as follows, but let's keep it as is in the interest of time. We could revisit that if the above requires fixing at some point however.